src/models/thread.ts

```diff
@@ -186,7 +185,7 @@ export class Thread extends TypedEventEmitter<EmittedEvents, EventHandlerMap> {
             this._currentUserParticipated = true;
         }
 
-        const isThreadReply = event.getRelation()?.rel_type === RelationType.Thread;
+        const isThreadReply = event.getRelation()?.rel_type === THREAD_RELATION_TYPE.name;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?